packages/blade/src/components/Table/TablePagination.web.tsx

```diff
@@ -136,7 +141,7 @@ const getPaginationButtons = ({
   };
 };
 
-const _TablePagination = ({
+const _TablePagination = <T extends { paginationType: TablePaginationType }>({
```

Comment:
Why do we need to accept a generic here?

Reply:
`onPageChange` & `totalItemCount` are optional when paginationType is client but mandatory when it is server